### PR TITLE
Add trademark acknowledgment statements

### PR DESCRIPTION
**Pull request by n8willis**
This is in the realm of cleanup and aligning better with real-world practice. It just adds trademark acknowledgements to the "Sources/References" section of the README, where OpenType and Unicode are cited.

Not yet sure if we need to add one of those for literally every company/product/project mentioned, since nominal references will just come up from time to time, but certainly it seems important to be clear that when the docs mention the outside standards, it's for informative referencing and stays appropriate.

Basically borrowing the wording from what I see used on other font-related standards like WOFF.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,21 +68,23 @@ other interesteed readers are encouraged to check the
 examine the [image-generation logs](/images/README.md) to identify
 issues seen in the inline images.
 
-### Sources
+### References
 
-These documents draw from the following existing resources:
+These documents cite from the following informative references:
```
Comment (mikeday) [inline README.md:73]: "cite the" rather than "cite from the"?

Comment (n8willis) [reply README.md:73]: Sounds good.
